extensions/amp-google-vrview-image/OWNERS.yaml

```diff
@@ -2,3 +2,4 @@
 # https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example.yaml
 
 - dvoytenko
+- ampproject/wg-ui-and-a11y
```

Comment:
Since `wg-ui-and-a11y` is taking over here, it's probably safe to remove `dvoytenko`; he'll still be an owner (since he's a root-level owner), but that way he won't be pulled into PRs in this directory.

Reply:
Removed